empirical_calibration/core.py

```diff
@@ -187,20 +192,19 @@ def calibrate(covariates: np.ndarray,
     beta_init = np.zeros(num_covariates + 1)
   elif objective == Objective.QUADRATIC:
     if baseline_weights_is_none:
-      weight_link = lambda x: np.clip(x, 0.0, max_weight)
+      weight_link = lambda x: np.clip(x, min_weight, max_weight)
       # Solution of the dual problem without the non-negative weight constraint.
-      beta_init = np.linalg.solve(
-          np.matmul(z.T, z), np.concatenate((np.ones(1),
-                                             np.zeros(num_covariates))))
+      # Use pseudoinverse in case z is not full rank.
+      beta_init = np.linalg.pinv(np.matmul(z.T, z)) @ np.concatenate(
```

Comment:
Could you please change `np.matmul` here and elsewhere into `@` for consistency?

Reply:
Done.